bin/analysercli

```diff
@@ -0,0 +1,276 @@
+#! /usr/bin/env python
+# -*- coding:utf-8 -*-
+#
+# This file is part of IVRE.
+# Copyright 2011 - 2015 Pierre LALET <pierre.lalet@cea.fr>
+#
+# IVRE is free software: you can redistribute it and/or modify it
+# under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# IVRE is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY
+# or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public
+# License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with IVRE. If not, see <http://www.gnu.org/licenses/>.
+
+from ivre import webutils
+from ivre.db import db
+from ivre.cliutils import COLORS, StackCli
+from ivre.analyser import Context
+
+
+class AnalyserCli(StackCli):
+    """
+    Command line utility wrapper to access analyser API
+    """
+    # COLORIZE levels
+    LOG_USAGE = 0
+    LOG_INFO = 10
+    LOG_WARNING = 20
```

Comment:
Maybe you could use a `dict` like:
```
LOG_LEVELS = {
    LOG_USAGE: {"name": "USAGE", "color": "green"},
    [...]
}
```

That could make the code in `_colorized_log()` better.

Also, what about moving that to the `cliutils` sub-module?